### PR TITLE
Logout

### PR DESCRIPTION
**Pull request by bhushannaikaroskar**
Add logout feature into products page. Please review code and suggest changes if any.
logout

### CODE REVIEW

**Review by saurabhan — APPROVED** [review]
clean code and readable, good use of utility classes.
approved

**Review by zoyatanweer — APPROVED** [review]
code is clean , good to merge

File: style.css [hunk 1]
```diff
@@ -312,4 +312,35 @@ body {
...
+    border: 1px solid var(--WHITE);
+    box-shadow: 0 1px 4px 0 var(--OFF-WHITE);
+}
+
+.account-modal::before{
+    content: "";
+    position: absolute;
+    top:-1.6rem;
+    right:0rem;
+    border-width: 15px;
```
Comment (srishtigautam21) [inline style.css]: Change to rem

**Review by srishtigautam21 — APPROVED** [review]
good to merge for now

**Pull request merged**